src/deepsparse/pipeline.py

```diff
@@ -220,8 +227,36 @@ def __call__(self, *args, **kwargs) -> BaseModel:
                 f"Outputs of {self.__class__} must be instances of "
                 f"{self.output_schema} found output of type {type(pipeline_outputs)}"
             )
+        timer.stop(InferencePhases.POST_PROCESS.value)
+        timer.stop(InferencePhases.TOTAL_INFERENCE.value)
+
+        if not monitoring:
+            return pipeline_outputs
+
+        else:
+            data = None
```

Comment:
let's return the input schema as well as engine inputs here. should not be pushing empty functionality to main